### PR TITLE
fix(json-schema): add option to report all errors (#63)

### PR DESCRIPTION
**Pull request by phb**

### CODE REVIEW

**Comment by jdanyow** [comment]
Changes looks great. 

One of the tests is failing:
```
  2366 passing (14s)
  1 failing

  1) draft2019-09/unevaluatedItems
       unevaluatedItems with anyOf
         when one schema matches and has unevaluted items:

      when one schema matches and has unevaluted items
      + expected - actual

      -true
      +false
```

Couple commands to run the tests for only the json-schema package:
```
yarn workspace @cfworker/json-schema test
yarn workspace @cfworker/json-schema test --watch --inspect
```

unevaluatedItems tests:
https://github.com/json-schema-org/JSON-Schema-Test-Suite/blob/master/tests/draft2019-09/unevaluatedItems.json#L205

**Comment by tsiq-johan** [comment]
argh, sloppy -- I was marking the new tests with .only and didn't run the whole testsuite. My apologies will fix and push an update.

**Comment by phb** [comment]
PR updated and should pass all tests now. Sorry for the noise (deleted previous comment made on my work account by mistake)

File: packages/json-schema/src/validate.ts [hunk 1]
```diff
@@ -630,7 +645,6 @@ export function validate(
             },
             ...result.errors
           );
-          break;
```
Comment (jdanyow) [inline packages/json-schema/src/validate.ts:633]: Would it be possible/better to put the `if (stop) break;` here to reduce the number of times that line of code executes?

Comment (tsiq-johan) [reply packages/json-schema/src/validate.ts:633]: Yep, agreed. Thanks for spotting

Comment (phb) [reply packages/json-schema/src/validate.ts:633]: Moved, thanks!

**Pull request merged**